source/slang/slang-check-impl.h

```diff
@@ -2980,4 +2993,36 @@ namespace Slang
         FrontEndEntryPointRequest* entryPointReq);
 
     bool resolveStageOfProfileWithEntryPoint(Profile& entryPointProfile, CompilerOptionSet& optionSet, const List<RefPtr<TargetRequest>>& targets, FuncDecl* entryPointFuncDecl, DiagnosticSink* sink);
+
+    LookupResult lookUpMember(
+        ASTBuilder* astBuilder,
+        SemanticsVisitor* semantics,
+        Name* name,
+        Type* type,
+        Scope* sourceScope,
+        LookupMask          mask,
+        LookupOptions       options);
+
+    ConstructorDecl* _getDefaultCtor(StructDecl* structDecl);
+    bool allParamHaveInitExpr(ConstructorDecl* ctor);
+    List<ConstructorDecl*> _getCtorList(ASTBuilder* m_astBuilder, SemanticsVisitor* visitor, StructDecl* structDecl, ConstructorDecl** defaultCtorOut);
+    bool DiagnoseIsAllowedInitExpr(VarDeclBase* varDecl, DiagnosticSink* sink);
+    bool isDefaultInitializable(Type* varDeclType, VarDeclBase* associatedDecl);
+    Expr* constructDefaultInitExprForVar(SemanticsVisitor* visitor, TypeExp varDeclType, VarDeclBase* decl);
+
+    enum class ConstructZeroInitListOptions : UInt
+    {
+        None = 0 << 0,
+        PreferZeroInitFunc = 1 << 0,
+        CheckToAvoidRecursion = 1 << 2,
+    };
+    Expr* constructZeroInitListFunc(SemanticsVisitor* visitor, StructDecl* structDecl, Type* structDeclType, ConstructZeroInitListOptions options);
+    FuncDecl* findZeroInitListFunc(StructDecl* structDecl);
+
+    bool checkIfCStyleStruct(SemanticsVisitor* visitor, StructDecl* decl);
```

Comment:
should be named `isCStyleStructDecl`.

Reply:
addressed here:
https://github.com/shader-slang/slang/pull/5107/commits/54014949789e700a589f5eea098443f3e2bc337c#diff-650283cec951a78df98dcededc742d1a4c434f3f2179ac2ab6dcbd29de9b5d4bR553